### PR TITLE
Default required

### PR DESCRIPTION
**Pull request by wizzat**
This adds the ability to set default required and nullable status.  It also adds the ability to override required and nullable status per array accessor.  It is built on top of PR #6, which adds Python 3 support.

### CODE REVIEW

**Comment by wizzat** [comment]
Awesome - thanks!

File: json_schema_generator/generator.py [hunk 1]
```diff
@@ -26,7 +36,7 @@ def from_json(cls, base_json):
 
         return obj
 
-    def to_dict(self, base_object=None, object_id=None, first_level=True):
+    def to_dict(self, base_object=None, object_id=None, first_level=True, required = True, nullable = False):
         """docstring for to_dict"""
```
Comment (justnpT) [inline json_schema_generator/generator.py]: Can we provide better for what arguments of the methods do in the future ?
Thanks :)

File: json_schema_generator/validator.py [hunk 2]
```diff
@@ -30,7 +30,7 @@ def assert_json(self, json_str):
             validate(json_object, self.json_schema_dict)
             valid = True
 
-        except Exception, e:
+        except Exception as e:
             self._error_message = "Inválido: \n\t%s" % str(e)
```
Comment (justnpT) [inline json_schema_generator/validator.py]: can we make that error handling english ?

File: json_schema_generator/generator.py (hunk 1 above)
Comment (wizzat) [inline json_schema_generator/generator.py]: It's a great idea, but I wasn't trying to bloat the diff by messing with existing doc strings.

File: json_schema_generator/validator.py (hunk 2 above)
Comment (justnpT) [inline json_schema_generator/validator.py]: Can we add the "path" so that it is stated what part of the response is invalid ?

**Pull request merged**